LICENSE-binary

```diff
@@ -269,11 +269,11 @@ See licenses-binary/ for text of these licenses.
     (Apache License, Version 2.0) Apache Directory LDAP API Utilities (org.apache.directory.api:api-util:1.0.0-M20 - http://directory.apache.org/api-parent/api-util/)
     (Apache License, Version 2.0) Apache Extras™ for Apache log4j™. (log4j:apache-log4j-extras:1.2.17 - http://logging.apache.org/log4j/extras)
     (Apache License, Version 2.0) Apache FreeMarker (org.freemarker:freemarker:2.3.29 - https://freemarker.apache.org/)
-    (Apache License, Version 2.0) Apache Hadoop Annotations (org.apache.hadoop:hadoop-annotations:2.6.5 - no url defined)
-    (Apache License, Version 2.0) Apache Hadoop Auth (org.apache.hadoop:hadoop-auth:2.6.5 - no url defined)
-    (Apache License, Version 2.0) Apache Hadoop Client (org.apache.hadoop:hadoop-client:2.6.5 - no url defined)
-    (Apache License, Version 2.0) Apache Hadoop Common (org.apache.hadoop:hadoop-common:2.6.5 - no url defined)
-    (Apache License, Version 2.0) Apache Hadoop HDFS (org.apache.hadoop:hadoop-hdfs:2.6.5 - no url defined)
+    (Apache License, Version 2.0) Apache Hadoop Annotations (org.apache.hadoop:hadoop-annotations:2.7.2 - no url defined)
+    (Apache License, Version 2.0) Apache Hadoop Auth (org.apache.hadoop:hadoop-auth::2.7.2 - no url defined)
+    (Apache License, Version 2.0) Apache Hadoop Client (org.apache.hadoop:hadoop-client::2.7.2 - no url defined)
```

Comment:
When the mvn license plugin cannot obtain the dependent project <project-url>, delete "no url defined" directly, or add the correct url of the project.